test/uk/gov/hmrc/servicemetrics/scheduler/MongoNotificationsSchedulerSpec.scala

```diff
@@ -160,6 +161,18 @@ class MongoNotificationsSchedulerSpec
         
       }
     }
+
+    "runs only during working hours" in new MongoNotificationsSchedulerFixture {
```

Comment:
You could do this with property based testing?

Reply:
I thought of it, I thought it be simple enough to not need it but can add it now :)